tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -663,6 +663,34 @@ def image_exists(self, imagespec):
 
         return 0
 
+    def rename_primary_image(self, source_imagespec, dest_imagespec, peercluster, **kw):
```

Comment:
We can have method named as rename_image (as a single instance of RbdMirror would be used to a single cluster)

Reply:
I will change it.